src/app/pages/lesson/dialogs/delete/delete-lesson.dialog.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, Inject } from '@angular/core';
```

Comment:
Did you happen to check if you can reuse other dialogs in the project? As far as I know this is not the first delete dialog

Reply:
all the examples is saw, the dialog had a dedicated text like '....delete class...' embedded in the dialoge. since this my first interaction with the code i just followed the current practice. once i get to know the code better i might have those kinks of suggestions :-)